vispy/color/colormap.py

```diff
@@ -1080,53 +1079,72 @@ def __init__(self, limits=(0.33, 0.66, 1.0)):
         ],
         color_space="hsv"
     )),
-    PuGr=_Diverging(145, 280, 0.85, 0.30),
-    GrBu=_Diverging(255, 133, 0.75, 0.6),
-    GrBu_d=_Diverging(255, 133, 0.75, 0.6, "dark"),
-    RdBu=_Diverging(220, 20, 0.75, 0.5),
+    PuGr=Diverging(145, 280, 0.85, 0.30),
+    GrBu=Diverging(255, 133, 0.75, 0.6),
+    GrBu_d=Diverging(255, 133, 0.75, 0.6, "dark"),
+    RdBu=Diverging(220, 20, 0.75, 0.5),
 
-    # Configurable colormaps
     cubehelix=CubeHelixColormap,
-    single_hue=_SingleHue,
-    hsl=_HSL,
-    husl=_HUSL,
-    hsluv=_HSLuv,
-    diverging=_Diverging,
-    RdYeBuCy=_RedYellowBlueCyan,
+    single_hue=SingleHue,
```

Comment:
What about adding the default `SingleHue` instance here (and same for classes below). That way we don't break things in terms of keys, but still remove the classes, which was the original goal.

Reply:
I think that's what I had originally. I guess that would still work if I update the deprecation warning to use `.__class__.__name__`. Then it also effects anyone using the dictionary originally. I'd still like the deprecation warning even though they would not be instances since they are probably not what people want/expect.